hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/ContainerEndpoint.java

```diff
@@ -627,39 +626,35 @@ public Response getContainerMisMatchInsights(
       case OM:
         List<ContainerInfo> nonOMContainers = new ArrayList<>();
         while (scmContainerInfo != null && nonOMContainers.size() < limit) {
-          Long omContainerID = containerMetadata == null ? null : containerMetadata.getContainerID();
+          Long omContainerID = omContainers.peekNextKey();
```

Comment:
OM case,
- sort scm removing deleted containers.
iterate scm,
   - check via seek() as current OR via isExist(), and identify which is better